packages/mui-material/src/styles/createPalette.d.ts

```diff
@@ -130,6 +130,24 @@ export interface PaletteWithChannels {
   getContrastText: (background: string) => string;
   augmentColor: (options: PaletteAugmentColorOptions) => PaletteColor;
   // component tokens
+  Alert: {
+    errorColor: string;
+    infoColor: string;
+    successColor: string;
+    warningColor: string;
+    errorFilledBg: string;
+    infoFilledBg: string;
+    successFilledBg: string;
+    warningFilledBg: string;
+    errorStandardBg: string;
+    infoStandardBg: string;
+    successStandardBg: string;
+    warningStandardBg: string;
+    errorIconColor: string;
+    infoIconColor: string;
+    successIconColor: string;
+    warningIconColor: string;
```

Comment:
@mnajdova I could not see other ways to reduce the number of tokens if we want to keep the same behavior. 😢

Reply:
Would be tricky if people add custom colors here :\ Should we maybe create these based on the PaletteOptions type?

Reply:
I don't think we should go that far for now. There is a trade-off because some people might not want the autogenerated variables (increase in bundle size) and we need to create more API to exclude them.

At this point, I think we can leave the new custom palette effort to the developers and make sure that the docs are clear and comprehensive.